Lua/bounty.lua

```diff
@@ -0,0 +1,35 @@
+-- Courtesy of Rei --
+
+-- Notes --
+-- Traitormod.AwardPoints(client, refundTable.Price)
+-- local points = Traitormod.GetPoints(client)
+
+local price = 500
+
+local killers = {
+    RName = {},
+    CName = {},
+    Count = {}
+}
+
+Hook.Add("character.death", "playerDeath", function (character)
+    local killerName = character.LastAttacker
+    if killerName.TeamID ~= CharacterTeamType.Team2 or killerName.TeamID ~= CharacterTeamType.Team0 then return end
+    
+    killers.RName[killerName] = killerName
+    killers.CName[killerName] = Util.FindClientCharacter(killerName)
+    killers.Count[killerName] += 1
+end)
+
+Traitormod.AddCommand("!claim", function (client, args) 
+    local found = nil
+    local bounty = nil
+    for i=1, killers.CName do
+        if killers.CName[i] == client.Name then
```

Comment:
if killers.CName[i] == client then

Reply:
this is because Util.FindClientCharacter(killerName) returns client and not client.Name